src/helpers/get-cosmos-min-amount.tsx

```diff
@@ -0,0 +1,13 @@
+import {Balance} from '@app/services/balance';
+import {COSMOS_MIN_AMOUNT} from '@app/services/cosmos';
+import {RemoteConfig} from '@app/services/remote-config';
+
+export const getCosmosMinAmount = () => {
```

Comment:
All of this helpers can be replaces by one generic

Reply:
Updated